templates/_call-stack-item.php

```diff
@@ -48,9 +56,38 @@
         </div>
 
         <?php if ($line !== null): ?>
-            <div><?= sprintf('at line %d', $line + 1) ?></div>
+            <div class="flex flex-column">
+                <?= sprintf('at line %d', $line + 1) ?>
+                <?php if (!empty($args)): ?>
+                    <button class="show-arguments-toggle">arguments</button>
+                <?php endif ?>
+            </div>
         <?php endif ?>
     </div>
+    <div class="function-arguments-wrap hidden">
+        <?php
+        if ($function !== null) {
+            echo '<table class="table w-100">';
+            foreach ($args as $key => $argument) {
+                echo '<tr class="argument">';
+                $key = is_int($key) && isset($reflectionParameters[$key]) ? $reflectionParameters[$key]->getName() : $key;
+                echo '<td>';
+                echo '<span class="argument-key bold">$' . $key . '</span>';
+                echo '</td>';
+                echo '<td>';
+                echo '<span class="argument-value word-break">';
+                echo $this->argumentsToString(is_array($argument) ? $argument : [$argument]);
```

Comment:
Should `$this->htmlEncode(` be added there?

Reply:
I think it's possible

Reply:
Let's do it then.

Reply:
There're no need to do it because it does inside the `argumentsToString`